old-fashioned-image-build

```diff
@@ -34,8 +34,8 @@ while :; do
             ;;
         --extra-ppa)
             # inject public ppas into the build environment
-            # e.g. 'lp:mylpname/someppa'
-            EXTRA_PPA="--extra-ppa \"$2\""
+            # e.g. 'mylpname/someppa'
+            EXTRA_PPA="--extra-ppa $2"
```

Comment:
You don't detail the exact failure you're seeing which makes this difficult to review.

It feels like adding literal quotes would be a deliberate attempt to avoid $IFS from incorrectly separating a ppa name into multiple arguments.  However I don't believe PPA names or LP user names can contain spaces or tabs, so I don't see how they could be split if they were left unquoted.

Basically, I think this is safe but I'd like to see the failure you logged to make me more comfortable with this.

Reply:
sorry, should've fleshed out the description more. It's been updated with error logs and reasoning.